engine/swarm/manage-nodes.md

```diff
@@ -184,6 +184,25 @@ Manager node-2 demoted in the swarm.
 `docker node update --role manager` and `docker node update --role worker`
 respectively.
 
+## Install plugins on swarm nodes
+
+{% include edge_only.md section="option" %}
+
+If your swarm service relies on one or more
+[plugins](/engine/extend/plugin_api/), these plugins need to be available on
+every node where the service could potentially be deployed. You can manually
+install the plugin on each node or script the installation. In Docker 17.07 and
+higher, you can also deploy the plugin as a global "service" using the Docker
```

Comment:
Just a note here, while it is a global service, it's not the same as `docker service create --global` which creates services that use the container runtime only.
This may be confusing to readers.